src/library_pthread.js

```diff
@@ -211,7 +211,7 @@ var LibraryPThread = {
 #endif
       // Call thread init functions (these are the emscripten_tls_init for each
       // module loaded.
-      for (var i in PThread.tlsInitFunctions) {
+      for (var i in PThread.tlsInitFunctions.keys()) {
```

Comment:
Actually it looks like the `PThread.tlsInitFunctions.hasOwnProperty(i))` is needed here instead.. if we want to continue to avoid `for .. of` (which I think we do for now).

Reply:
Alright, updated the PR to check that before calling the function 👍